ios/RNViewShot.m

```diff
@@ -41,7 +47,14 @@ - (dispatch_queue_t)methodQueue
 
     // Get view
     UIView *view;
-    view = viewRegistry[target];
+
+    if ((int)target == -1) {
```

Comment:
this does not work, I think because target is a `NSNumber *` pointer. you need instead to do: `[target intValue]`